app/SearchApi/SearchApi.Web/Startup.cs

```diff
@@ -18,6 +18,8 @@
 using OpenTracing.Util;
 using SearchApi.Core.Configuration;
 using SearchApi.Core.Contracts;
+using SearchApi.Core.MassTransit;
```

Comment:
Is this import needed?

Reply:
yes for the extensions methods to register MassTransit Service Bus